.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
 bundle.js
+.cache/
+package-lock.json
```

Comment:
this is not a good thing to add... package-lock.json should always be committed.

Reply:
Apologies, will revert.

Reply:
Apologies, will revert.